src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/util/OcrDataDeserializerTest.java

```diff
@@ -21,30 +29,19 @@
 
     @Test
     public void should_convert_all_fields_to_their_string_representation() throws Exception {
-        Map<String, String> result = deserializeFromBase64("/ocr-data/valid/valid-ocr.json");
-
-        Map<String, String> expectedResult = ImmutableMap.of(
-            "text_field", "some text",
-            "number_field", "123",
-            "boolean_field", "true",
-            "null_field", ""
+        OcrData resultOcrData = deserializeFromBase64("/ocr-data/valid/valid-ocr.json");
+
+        OcrData expectedOcrData = new OcrData();
+        List<OcrDataField> ocrDataFields = Arrays.asList(
+            createOcrDataField(new TextNode("text_field"), new TextNode("some text")),
+            createOcrDataField(new TextNode("number_field"), new IntNode(123)),
+            createOcrDataField(new TextNode("boolean_field"), BooleanNode.TRUE),
+            createOcrDataField(new TextNode("null_field"), NullNode.instance)
         );
 
-        assertThat(result).isEqualTo(expectedResult);
-    }
+        expectedOcrData.setFields(ocrDataFields);
```

Comment:
Not strictly related with this PR, but I wonder if OcrData could be made immutable.